packages/plugin-ext/src/main/browser/plugin-contribution-handler.ts

```diff
@@ -136,11 +136,17 @@ export class PluginContributionHandler {
         if (contributions.viewsContainers) {
```

Comment:
register commands and menu items before other contributions in order to make sure that they are present when view containers are created